docs/config.md

```diff
@@ -91,6 +76,13 @@ CommaSeparatedStrings(['127.0.0.1', 'localhost'])
 '127.0.0.1'
 ```
 
+!!! tip
+
+    You can use `DatabaseURL` from `databases`
+    package [here](https://github.com/encode/databases/blob/ab5eb718a78a27afe18775754e9c0fa2ad9cd211/databases/core.py#L420)
+    to store database URLs and avoid leaking them in the logs.
```

Comment:
This is supposed to be in the section above, isn't it right? 🤔

Reply:
I was thinking it should go after Secret and CommaSeparatedStrings (Starlette's types) but it can also go immediately after Secret which is more related.